### DISCUSSION TITLE
[Guild Suggestion] Guild Scaled GEXP Into response (/g info) | [Guild Issue] Incorrect values/dates being returned

### DISCUSSION THREAD

**Discussion by ugcodrr** [HypixelDev/PublicAPI#237] [Ideas]
![image](https://user-images.githubusercontent.com/36463298/81456846-ceb71800-9148-11ea-8957-b4690eee2d5e.png) (Title)

**Yes**, I know it is possible to add the total of every member's gexp for that day HOWEVER with the api returning incorrect/false data (see #204 &#145) and guild members getting kicked/leaving it is not possible to create an accurate remake of the command.

**Comment by SimplyNo** [comment]
yes please

**Comment by Kuba314** [comment]
The only fix to this that I can think of is to check whether the size of all members' `expHistory` objects is 7 and if not discard the result. Repeat until true :))
